### ISSUE TITLE
Elevated plasma linoleylcarnitine, C18:2

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6425]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated Linoleylcarnitine concentration in the blood.

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
